cdap-docs/build.sh

```diff
@@ -171,18 +174,23 @@ function build_docs_outer_level() {
 ################################################## current
 
 function build_all() {
+  echo "========================================================================="
   echo "Building GitHub Docs."
-  ./build.sh docs-github $ARG_2 $ARG_3
+  echo "========================================================================="
+  run_command docs-github-part $ARG_2 $ARG_3
   echo "Stashing GitHub Docs."
   cd $SCRIPT_PATH
   mkdir -p $SCRIPT_PATH/$BUILD_TEMP
   mv $SCRIPT_PATH/$BUILD/*.zip $SCRIPT_PATH/$BUILD_TEMP
+  echo "========================================================================="
   echo "Building Web Docs."
-  ./build.sh docs-web $ARG_2 $ARG_3
+  echo "========================================================================="
+  run_command docs-web-part $ARG_2 $ARG_3
   echo "Replacing GitHub Docs."
   mv $SCRIPT_PATH/$BUILD_TEMP/*.zip $SCRIPT_PATH/$BUILD
   rm -rf $SCRIPT_PATH/$BUILD_TEMP
   bell
+  exit 1
```

Comment:
I think the reason why I did it this is so that you can cancel a build in progress. Because when you do a control-c, if can get grabbed by Java or maven or Sphinx (python), and IKBW, but I think it was necessary to get the bigger script to not keep going. Plus, there are invocations inside invocations: the lower-level doc builds are run from inner shell scripts.